.github/CHECKLIST.md

```diff
@@ -0,0 +1,34 @@
+Checklists for Pull Requests and Releases
+=========================================
+
+Maintainer checklists for reviewing pull requests and doing releases.
+
+
+Pull Requests
+-------------
+
+ - If applicable, is there a readable ChangeLog entry?
+ - If any LICENSE file has been updated, has the `.hash` file been updated?
+ - If any change to a Finit `.svc` file, does any run/task it linger?  
```

Comment:
```suggestion
 - If any change to a Finit `.svc` file, does any run/task linger?  
```
Maybe? Not sure how it relates to the sentence below.

Reply:
Thanks, that was a bit unclear.